README.md

```diff
@@ -11,69 +11,54 @@ See [what's deployed](https://whatsdeployed.io/s-J18)
 
 ## Development
 
-To setup a local Kitsune development environment:
+To get Kitsune running locally all you really need is to have [Docker installed](https://www.docker.com/products/docker-desktop),
+and follow the following steps.
 
 1. Fork this repository & clone it to your local machine.
-
-1. Download base Kitsune docker images:
    ```
-   docker pull mozmeao/kitsune:base-latest
-   docker pull mozmeao/kitsune:base-dev-latest
+   git clone https://github.com/mozilla/kitsune.git
    ```
 
-1. Build Kitsune docker images. (Only needed on initial build or when packages change)
+2. Pull base Kitsune Docker images, create your database, and install node and bower packages.
    ```
-   docker-compose -f docker-compose.yml -f docker/composefiles/build.yml build base
-   docker-compose -f docker-compose.yml -f docker/composefiles/build.yml build dev
+   make init
    ```
 
-1. Copy .env-dist to .env
+3. Run Kitsune.
    ```
-   cp .env-dist .env
+   make run
```

Comment:
There's a lot of output from the commands listed on this page, some success output might be nice to guide the user:

For example:

The server is ready if you see this:

```
web_1              | System check identified no issues (3 silenced).
web_1              | November 09, 2018 - 06:44:42
web_1              | Django version 1.8.15, using settings 'kitsune.settings'
web_1              | Starting development server at http://0.0.0.0:8000/
web_1              | Quit the server with CONTROL-C.
```

Reply:
That's a good point.